src/actions/accountActions.ts

```diff
@@ -17,16 +17,30 @@
 import type { Account } from '../types/accountTypes';
 
 enum AccountActionsTypes {
-  SET_ACCOUNT = 'SET_ACCOUNT'
+  SET_ACCOUNT = 'SET_ACCOUNT',
+  SET_DERIVED_ACCOUNT = 'SET_DERIVED_ACCOUNT ',
```

Comment:
are the spaces at the end of each string (SET_DERIVED_ACCOUNT and SET_GENERIC_ACCOUNT intentional? looks like a typo